organizations.json

```diff
@@ -349,9 +345,8 @@
         "city": "Curitiba, PR",
         "latitude": "-25.4952",
         "longitude": "-49.2874",
-        "type": "Code for All",
         "tags": [
-            "Code for All"
+          "Brigade"
```

Comment:
We're doing a 4-space indent in this file. You can have all the formatting fixed for you automatically with:

```
bin/format-json organizations.json
```

You might need to `brew install jq moreutils` first before that script will run.